data-reconciliation-app/src/endpoints/ingest.ts

```diff
@@ -0,0 +1,32 @@
+import * as ccfapp from "@microsoft/ccf-app";
+import { ServiceResult } from "../utils/service-result";
+import { ingestService, authenticationService } from "../utils/dependencies";
+import { ApiResult } from "../utils/api-result";
+import { DataSchema } from "../models/data-schema";
+
+export function submitData(request: ccfapp.Request<any>): ccfapp.Response<any> {
```

Comment:
The response type can be `ApiResult<T>` where T can be an interface returned from `ingestService.submitData`.

```typescript
export interface SubmitDataResponse {
 //////
}